lib/cpanfile.pod

```diff
@@ -23,6 +23,9 @@ cpanfile - A format for describing CPAN dependencies for Perl applications
     recommends 'DBD::SQLite';
   };
 
+  requires 'App::cpanminus',
```

Comment:
note that this would be inconsistent with the format as defined on line 44:

```
requires $module, $version_requirement, %options;
```

(although that line is already a bit inaccurate, plenty of `cpanfile` examples in the wild with no `$version_requirement` provided...)

Reply:
In the description: "When version requirement is omitted, it is assumed that 0 is passed, meaning any version of the module would satisfy the requirement."

Reply:
Yes, I see the description, and it works well enough ("a bit inaccurate", not "completely wrong") if the `$version_requirement` is last in the list... anyone implementing just needs `my ($module, $version_requirement) = @_; $version_requirement //= 0;`, which is standard enough as Perl constructs go.

```
requires $module, $version_requirement, %options;
```

Two scalars, followed by a list of key => value pairs. Now sometimes one of those scalars isn't a scalar? How would you write that documentation line if it takes an "optional" version scalar?

```
requires $module, @version_requirement_list_with_one_or_zero_items, %options;
```

?

Looks like it'd drag in needless complexity for anyone implementing this or even trying to understand the documentation from the perspective of a Perl programmer who's maybe seen a scalar or two in their time: far simpler to say "if `%options` are passed, make the version mandatory".

Reply:
The options are required to be in pairs, so you just count the number of arguments.